_data/library/jsfacades.yml

```diff
@@ -66,8 +66,20 @@
       dep: '"org.singlespaced" %%% "scalajs-d3" % "0.3.4"'
     - name: scaladget
       url: https://github.com/mathieuleclaire/scaladget
-      desc: Static types for [D3.js](http://d3js.org), [Ace](http://ace.c9.io),  [Bootstrap-native.js](https://thednp.github.io/bootstrap.native/) (and an API for the latter)
-      dep: '"fr.iscpif" %%% "scaladget" % "0.9.2"'
+      desc: Static types for [Ace](http://ace.c9.io).
+      dep: '"fr.iscpif.scaladget" %%% "ace" % "1.0.1"'
```

Comment:
There is already a facade for `ace` in this file. Consider moving this definition next to the other one.